code/src/components/AllIcons.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+import HeartIcon from '../Icons/heart.png'
+import PlayIcon from '../Icons/play.png'
+import EllipsisIcon from '../Icons/ellipsis.png'
+
+
+export const AllIcons = () => {
+    return (
+        <div className="overlay">
+            <img className="icon-heart icons" src={HeartIcon} alt="A heart icon that works as a button" />
+            <img className="icon-play icons" src={PlayIcon} alt="The icon of a play button" />
+            <img className="icon-ellipsis icons" src={EllipsisIcon} alt="An elipsis icon that works as a button" />
```

Comment:
To make the site more accessible, consider changing the alt text on the icons since they do not work as buttons. Alternatively, have them link to the album maybe so they work as buttons :)